pod/perlre.pod

```diff
@@ -1152,13 +1152,7 @@ string:
     print; 	# prints 1234X6789, not XXXXX6789
 
 
-=head3 Capture groups
-
-The grouping construct C<( ... )> creates capture groups (also referred to as
-capture buffers). To refer to the current contents of a group later on, within
-the same pattern, use C<\g1> (or C<\g{1}>) for the first, C<\g2> (or C<\g{2}>)
-for the second, and so on.
-This is called a I<backreference>.
+=head3 Capture Groups and Backreferences
```

Comment:
This would also break existing links to this section and I'm not sure how much value it brings to add the word backreferences to the heading.

Reply:
This is related to the other point I am parking for now. I want to get the other stuff cleaned up and then ill thinking about further restructuring.